systemvm/agent/noVNC/core/rfb.js

```diff
@@ -2608,6 +2656,7 @@ export default class RFB extends EventTargetMixin {
                 return true;
 
             case encodings.pseudoEncodingVMwareCursor:
+                this._VMwareCursorSupported = true;
```

Comment:
Is this encoding a requirement to display the JP keyboard?

Reply:
this is just to make sure that vm is running on vmware env.
it is probably unnecessary, as we use  `requiresVncOverWebSocketConnection` in java class